client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -43,26 +44,76 @@ class Transfer extends React.PureComponent {
 		this.props.paymentSettingsClick( this.props.domain );
 	};
 
+	restartTransfer = () => {
+		const { domain, selectedSite, translate } = this.props;
+
+		restartInboundTransfer( selectedSite.ID, domain.name, ( error, result ) => {
+			if ( result ) {
+				this.props.successNotice( translate( 'The transfer has been successfully restarted.' ), {
+					duration: 5000,
+				} );
+				fetchDomains( selectedSite.ID );
+			} else {
+				this.props.errorNotice(
+					error.message || translate( 'We were unable to restart the transfer.' ),
+					{
+						duration: 5000,
+					}
+				);
+			}
+		} );
+	};
+
+	getCancelledContent() {
+		const { domain, translate } = this.props;
+
+		return (
+			<Card>
+				<div>
+					<p className="edit__transfer_text_fail">{ translate( 'Domain Transfer Failed' ) }</p>
```

Comment:
You should change the `className` to `edit__transfer-text-fail` - the underscores are the reason CI fails

Reply:
:man_facepalming: